packages/components/src/popover/index.js

```diff
@@ -162,25 +162,26 @@ const Popover = (
 		: placement;
 
 	const ownerDocument = useMemo( () => {
+		let documentToReturn;
```

Comment:
These changes are temporarily applied to this PR, while waiting for #42886 to get merged

Reply:
https://github.com/WordPress/gutenberg/pull/42886 got merged, and I removed those changes